Cycling Analysis/Strava importer.ipynb

```diff
@@ -416,7 +427,7 @@
     "durations = [5, 10, 30, 60, 5*60, 10*60, 20*60, 30*60, 60*60, 90*60, 120*60]  # 5s, 10s, 30s, 60s, and 5min\n",
```

Comment:
In place of hardcoding durations, consider defining them as constants or configurable parameters.

Reply:
Hi @pranav5722222, these durations are fixed and are not expected to be configured. In fact, if they were configurable and the configuration was changed then it would invalidate all of the predictive models.